junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/HttpDataAssert.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit5.client;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.nio.charset.Charset;
+
+import com.linecorp.armeria.common.HttpData;
+
+/**
+ * Assertion methods for HttpData.
+ */
+public final class HttpDataAssert extends AbstractResponseAssert<HttpData> {
```

Comment:
While looking through `HttpDataAssert`, I felt a lot of the logic can be better defined/supplemented using `assertj` (i.e. `AbstractByteArrayAssert`).

Would it be worth just creating a separate `assertj` module for this purpose?

Reply:
Nevermind this comment 😅   Talked with the team and we decided not to move this to a separate module for this iteration